### PR TITLE
doc: create/update README.mds for examples

### PR DESCRIPTION
**Pull request by husayt**
## Types of changes

- [ ] Bug fix (a non-breaking change which fixes an issue)
- [ ] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Description

Added missing Readme

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly. (PR: #)
- [ ] I have added tests to cover my changes (if not applicable, please state why)
- [ ] All new and existing tests are passing.

### CODE REVIEW

**Comment by husayt** [comment]
Sorry if too many commits, wanted to finish while started. Some very useful examples were hidden without description.

File: examples/with-vuikit/README.md [hunk 1]
```diff
@@ -1,3 +1,3 @@
-# Nuxt with Vuikit
+# Nuxt with [Vuikit](https://vuikit.js.org)
 
-https://vuikit.js.org
+> Vuikit ia a consistent and responsive Vue UI library, based on the front-end framework UIkit.
```
Comment (manniL) [inline examples/with-vuikit/README.md]:
```suggestion
> Vuikit is a consistent and responsive Vue UI library, based on the front-end framework UIkit.
```

File: examples/with-firebase/README.md [hunk 2]
```diff
@@ -1,6 +1,4 @@
-# nuxt-firebase
-
-> Nuxt.js with Firebase (REST API)
+# Nuxt.js with [Firebase](https://firebase.google.com/) (REST API)
```
Comment (manniL) [inline examples/with-firebase/README.md]:
```suggestion
# Nuxt with [Firebase](https://firebase.google.com/) (REST API)
```

File: examples/auth-routes/README.md [hunk 3]
```diff
@@ -1,3 +1,3 @@
-# Authenticated Routes with Nuxt.js
+# Authenticated Routes with Nuxt.js & Express sessions
```
Comment (manniL) [inline examples/auth-routes/README.md]:
```suggestion
# Authenticated Routes with Nuxt and Express sessions
```

File: examples/middleware/README.md [hunk 4]
```diff
@@ -1,3 +1,6 @@
 # Middleware with Nuxt.js
```
Comment (manniL) [inline examples/middleware/README.md]:
```suggestion
# Middleware with Nuxt
```

File: examples/vuex-persistedstate/README.md [hunk 5]
```diff
@@ -1,3 +1,4 @@
-# Nuxt.js with Vuex persisted state (localStorage)
+# Nuxt.js with [Vuex](https://vuex.vuejs.org/) persisted state (localStorage)
```
Comment (manniL) [inline examples/vuex-persistedstate/README.md]:
```suggestion
# Nuxt with [Vuex](https://vuex.vuejs.org/) persisted state (localStorage)
```

File: examples/vuex-store-modules/README.md [hunk 6]
```diff
@@ -1,3 +1,7 @@
-# Nuxt.js with Vuex via File API
+# Nuxt.js with [Vuex](https://vuex.vuejs.org/) and store modules
```
Comment (manniL) [inline examples/vuex-store-modules/README.md]:
```suggestion
# Nuxt with [Vuex](https://vuex.vuejs.org/) and store modules
```

File: examples/vuex-store/README.md [hunk 7]
```diff
@@ -1,3 +1,7 @@
-# Nuxt.js with Vuex
+# Nuxt.js with [Vuex](https://vuex.vuejs.org/)
```
Comment (manniL) [inline examples/vuex-store/README.md]:
```suggestion
# Nuxt with [Vuex](https://vuex.vuejs.org/)
```

File: examples/with-cookies/README.md [hunk 8]
```diff
@@ -1,3 +1,5 @@
-# Hello World with Nuxt.js
+# Nuxt.js and cookies example
```
Comment (manniL) [inline examples/with-cookies/README.md]:
```suggestion
# Nuxt with cookies
```
Maybe?

File: examples/with-feathers/README.md [hunk 9]
```diff
@@ -1,6 +1,4 @@
-# with-feathers
-
-> Nuxt.js with Feathers
+# Nuxt.js with [Feathers](http://feathersjs.com)
```
Comment (manniL) [inline examples/with-feathers/README.md]:
```suggestion
# Nuxt with [Feathers](http://feathersjs.com)
```

File: examples/with-element-ui/README.md [hunk 10]
```diff
@@ -0,0 +1,3 @@
+# Nuxt.js with [Element UI](https://element.eleme.io/#/en-US)
```
Comment (manniL) [inline examples/with-element-ui/README.md]:
```suggestion
# Nuxt with [Element UI](https://element.eleme.io/#/en-US)
```

**Comment by manniL** [comment]
Thanks for the effort 🙏

**Comment by husayt** [comment]
@manniL Thanks for a detailed review and corrections

**Review by atinux — APPROVED** [review]
Nice improvements @husayt :)

**Pull request merged**